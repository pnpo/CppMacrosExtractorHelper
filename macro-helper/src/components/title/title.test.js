import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import { Title } from './index';

describe('Title test suite', () => {

    let wrapper, component;

    beforeEach(() => {

        wrapper = shallow(<Title />);

        component = renderer.create(
            <Title />
        );
    });

    it('render without title', () => {

        const json = component.toJSON();

        expect(json).toMatchSnapshot();
    });

    it('render with title', () => {

        const title = 'cx-web-kisckstart!';

        component = renderer.create(
            <Title title={title} />
        );

        const json = component.toJSON();

        expect(json).toMatchSnapshot();
    });

});
