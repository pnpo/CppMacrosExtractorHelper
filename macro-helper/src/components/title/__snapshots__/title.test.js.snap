// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title test suite render with title 1`] = `
<h1
  className="portal-title"
>
  cx-web-kisckstart!
</h1>
`;

exports[`Title test suite render without title 1`] = `
<h1
  className="portal-title"
/>
`;
